assets/styles/css/styles.scss

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@500;800;900&display=swap");
+@import "../css/nullstyle.css";
```

Comment:
It can be imported in HEAD before importing main styles, try using one imports style